package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "next": "^12.0.4",
     "react": "17.0.2",
+    "react-countdown": "^2.3.2",
```

Comment:
Countdown timer component: https://www.npmjs.com/package/react-countdown

Reply:
In the future, we can try to update this to use a timer we build ourselves, but I chose to focus on the game dynamic here, since timers are a pain in my experience.